client/js/lounge.js

```diff
@@ -227,6 +227,10 @@ $(function() {
 		var chan = chat.find(target);
 		var msg;
 
+		if (highlights.some((h) => data.msg.text.indexOf(h) > -1)) {
```

Comment:
I fully agree with that, @YaManicKill. Just not worth breaking browsers for a couple fat arrow functions just now :-)